.reposync.yml

```
@@ -0,0 +1,2 @@
+exclusions:
+- src/NServiceBus.snk
```

This repo doesn't do strong naming, so that means we don't need the strong name key for test projects to use in the [InternalsVisibleTo attribute](https://github.com/Particular/NServiceBus.Storage.MongoDB/blob/backport-reposync-to-release/src/NServiceBus.Storage.MongoDB/InternalsVisibleTo.cs), which is already able to give NServiceBus.Storage.MongoDB.Tests access to methods marked internal just with the name of the project, because the MongoDB persistence itself isn't strong named, because it can't be due to the MongoDB client driver package being unsigned.
```suggestion
- src/NServiceBus.snk
- src/NServiceBusTests.snk
```